code-challenges.test.js

```diff
@@ -32,19 +32,83 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+// First, I have to start with a describe statement with a string and function 
+// I am going to name the test "divisibleBy" because I really don't have a more creative name that that to be honest
+// In my it statement I'm going to copy the end of the challenge to explain whats going to happen or supposed to happen in this test 
+// Because every number is NOT always divisible by 3 I put 2 expects, one when the number is divisible by 3 and one when the number isn't divisible by 3
+
+
+describe("divisibleBy", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+      expect(divisibleBy (object1, object2)).toEqual(`15 and 0 are divisibleBy 3`)
+      expect(divisibleBy (object3)).toEqual(`-7 is not divisibleBy 3`)
+    })
+  })
+
+// ReferenceError: num1 is not defined
+
 // b) Create the function that makes the test pass.
+// Now I will be making the function to define divisibleBy 
+// A conditional statement will do the job 
```

Comment:
I would HIGHLY recommend expanding on your pseudocode for your function.  This is where all of the main logic comes together and where things can get tricky.